docs/BUILD.md

```diff
@@ -71,16 +71,16 @@ repo the following command is used:
 
 The `-SNAPSHOT` suffix is mandatory. 
 
-**Note**: for publishing to OSS Sonatype you need specify **API** User/Key (not login credentials) for  
-`OSS_USER`/`OSS_KEY` environment variables or `ossUser`/`ossKey` gradle build properties 
+**Note**: for publishing to OSS Sonatype you need to specify **API** User/Key (not login credentials) for  
+`OSS_USER`/`OSS_KEY` environment variables or `ossUser`/`ossKey` Gradle build properties 
 (the properties have precedence over environment variables). 
 
 See more configuration details of the oss uploading in [oss-publish.gradle](/gradle-scripts/oss-publish.gradle) file.
 
 
 ## Publish to Bintray
 
-[Bintray documentation about publish process](https://blog.bintray.com/2014/02/11/bintray-as-pain-free-gateway-to-maven-central/)
+[Bintray documentation about publishing process](https://blog.bintray.com/2014/02/11/bintray-as-pain-free-gateway-to-maven-central/)
```

Comment:
maybe `the publishing`?

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/319/commits/2267cb8153105dba51b128821406a039e2d931c6